services/ec2/drivers/Ec2EbsSnapshot.py

```diff
@@ -26,4 +27,19 @@ def _checkSnapshotPublic(self):
         if len(publicSnapshotList) > 0:
             self.results['EBSSnapshotIsPublic'] = [-1, ', '.join(publicSnapshotList)]
             
+        return
+
+    def _checkDeletedVolumeSnapshotList(self):
+        snapshotData = self.ec2Client.describe_snapshots(
+            OwnerIds = ["self"],
+        )
+
+        deletedVolumeSnapshotList = []
+        for snapshot in snapshotData['Snapshots']:
+            if snapshot['VolumeId'] not in self.ebsVolumeIds:
+                deletedVolumeSnapshotList.append(snapshot['SnapshotId'])
+        
+        if len(deletedVolumeSnapshotList) > 0:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To check if a container or sequence (string, list, tuple) is empty, use `if not val`. Do not compare its length using `if len(val) == 0`

[Learn more](https://www.python.org/dev/peps/pep-0008/#programming-recommendations#:~:text=if%20not%20seq)